test/fixtures/user/migration.js

```diff
@@ -0,0 +1,40 @@
+// Import fixtures
+const githubUserInfo = require("../auth/githubUserInfo")();
+
+/**
+ * User info for GitHub auth response
+ * Multiple responses can be added to the array if required
+ *
+ * @return {Object}
+ */
+module.exports = () => {
+  return [
+    {
+      username: "ankur",
+      first_name: "Ankur",
+      last_name: "Narkhede",
+      yoe: 0,
+      img: "./img.png",
+      linkedin_id: "ankurnarkhede",
+      github_user_id: githubUserInfo[0].id,
```

Comment:
this field shouldn't exist before migration. It will get added post migration. what do you think ?

Reply:
We're not using it anyway before migration so it's on us whether to keep it or not.

Reply:
but we will not be sure if it was updated by the migration api or not. so it would be recommended that we remove this before migrate.

Reply:
Removed the `github_user_id field`